meta-lxatac-software/recipes-core/bundles/files/hook.sh

```diff
@@ -44,7 +44,22 @@ function migrate () {
 	cp -a "$1" "${RAUC_SLOT_MOUNT_POINT}/$1"
 }
 
+function mock_resize2fs () {
+	# "old" bundles (~2023-10) do not have a resize2fs,
+	# making the resize phase of the bundle install fail.
+	# Provide a "resize2fs" so the installation works at least.
+	# Can be removed once all TACs are updated to have resize2fs.
+	if [ ! -f "/sbin/resize2fs" ]; then
+		echo "#!/bin/sh" > /sbin/resize2fs
+		echo "echo \"Dummy resize2fs installed by RAUC hook\"" >> /sbin/resize2fs
+		chmod +x /sbin/resize2fs
+	fi
+}
+
 case "$1" in
+	slot-pre-install)
+		mock_resize2fs
+		;;
```

Comment:
This does not work for us as RAUC tries to mount the old slot before running the `pre-install` hook.
On freshly deployed TACs there is however no partition to mount and the installation fails 😒.

Reply:
Nevermind. I misunderstood how the settings in in `system.conf` affect the generated bundle (_not at all_).
The generated bundles install just fine on existing devices (without resizing the installed rootfs though) and subsequent installs will have a rootfs that is grown to fit.